source/recipe.rst

```diff
@@ -308,66 +235,91 @@ The following properties are supported, with the mean specified by the
 property name, and the standard deviation by appending ``SD`` to the
 property name:
 
-- `gsyn`, the peak conductance (in nS) for a single synaptic contact, following a Gamma distribution
-- `d`, time constant (in ms) for recovery from depression, following a Gamma distribution
-- `f`, time constant (in ms) for recovery from facilitation, following a Gamma distribution
-- `u`, utilization of synaptic efficacy, following a truncated Normal distribution
-- `dtc`, decay time constant (in ms), following a truncated Normal distribution
-- `nrrp`, number of vesicles in readily releasable pool, following a Poisson distribution
+.. table::
+
+   ==================================== =========== ===
+   Property                             Requirement Description
+   ==================================== =========== ===
+   ``gsyn``                             Mandatory   The peak conductance (in nS) for a single synaptic contact, following a Gamma distribution.
```

Comment:
would it make sense for these to match the names in SONATA?

Reply:
it for sure would be less confusing and remove another layer of mapping

Reply:
Do you think we're (kinda?) making a new format, that is our chance to change it?

Reply:
Yes, will work on that

Reply:
@eleftherioszisis has noticed that the names are also used in NEXUS; maybe hold off a bit to see how much many changes that will require.

Reply:
I would also prefer if we all used the SONATA spec as our one and only truth. :)

Reply:
Based on @eleftherioszisis' feedback I've started on the renaming - it will make things less confusing and more straightforward within the functionalizer, and introduces a 1:1 correspondence between the recipe and the output, so that users will not have to wonder which settings affect which output column.